explore/src/main/scala/explore/Telemetry.scala

```diff
@@ -95,7 +95,7 @@ object Observability:
       .withHooks[Props]
       .useEffectOnMountBy: props =>
         val sdk = new HoneycombWebSDK(props.options)
-        Callback { println(sdk); sdk.start() } *> CallbackTo(
+        Callback.log(sdk) *> Callback(sdk.start()) *> CallbackTo(
```

Comment:
I guess we should remove the log

Reply:
I didn't know if you wanted to leave it, the way it was before it just printed `object Object`. I'll remove.